frontend/public/components/routes.tsx

```diff
@@ -130,52 +131,6 @@ export const RouteStatus: React.SFC<RouteStatusProps> = ({obj: route}) => {
 };
 RouteStatus.displayName = 'RouteStatus';
 
-const addTLSWarnings = (route: any, warnings: string[]) => {
-  if (!route.spec || !route.spec.tls) {
-    return;
-  }
-
-  if (!route.spec.tls.termination) {
-    warnings.push('Route has a TLS configuration, but no TLS termination type is specified. TLS will not be enabled until a termination type is set.');
-  }
-
-  if (route.spec.tls.termination === 'passthrough' && route.spec.path) {
-    warnings.push(`Route path "${ route.spec.path }" will be ignored since the route uses passthrough termination.`);
-  }
-};
-
-const addIngressWarnings = (route: any, warnings: string[]) => {
-  if (!route.status) {
-    return;
-  }
-
-  _.each(route.status.ingress, (ingress) => {
-    const condition: any = _.find(ingress.conditions, { type: 'Admitted', status: 'False' });
-    if (condition) {
-      let message = `Requested host '${ ingress.host || '<unknown host>' }' was rejected by the router.`;
-      if (condition.message || condition.reason) {
-        message += ` Reason: ${ condition.message || condition.reason }.`;
-      }
-      warnings.push(message);
-    }
-  });
-};
-
-export const RouteWarnings: React.SFC<RouteHostnameProps> = ({obj: route}) => {
-  const warnings: string[] = [];
-
-  if (!route) {
-    return;
-  }
-
-  addTLSWarnings(route, warnings);
-  addIngressWarnings(route, warnings);
-  return <React.Fragment>
-    {warnings.map((warning, index) => <div key={index} className="co-m-route-warnings">{warning}</div>)}
```

Comment:
Can you remove this class if it's no longer used?

Reply:
removed

Reply:
Our linting checks this correct?  If not, we should look into that, would rather have tooling tell us what is/is not used.